builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -242,6 +255,121 @@ node {
 
 }
 
+def getAssetsList(asset_url, current_environment_id) {
```

Comment:
would be nice to add some comments above the function (explaining what its doing)

Reply:
Done